Update mobile view

pranavkarthik10 opened the pull request:
<!-- Please read and understand everything below
**Do not delete any text other than where you are instructed to.** -->

<!-- **Students: If one of them is applicable to you. Please check it.** -->

<!-- Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.-->

- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request
<!-- If your change will be reflected on the website, please provide a Test Link-->
- [Preview Link](https://pranavkarthik10.github.io/2019.fossasia.org)
<!-- If you fully fixed some issue, please insert the issue number after the #.
If you have not completely fixed an issue but only some part of it, then remove “Fixes #” and simply link the PR to the issue by writing '#<Issue Number>' -->
- Fixes #118 


## Description / Changes
I updated the view using the overflow-x attribute for the body to hidden which allowed all overflow to be modified and changed to only fit the view.

## Screenshots if any:
### Before:
<img width="424" alt="Screen Shot 2019-12-29 at 3 43 30 PM" src="https://user-images.githubusercontent.com/44630385/71564215-fcc97480-2a51-11ea-8e60-ad309d2388a0.png">

### After:
<img width="424" alt="Screen Shot 2019-12-29 at 3 19 15 PM" src="https://user-images.githubusercontent.com/44630385/71564274-de17ad80-2a52-11ea-80ae-519c206b37f2.png">

- - - - - - - - - - - -
<!-- [preview link url]: https://<github_username>.github.io/<name_of_repository> -->
<!-- [squash]:https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git-->
<!--[squash2]https://davidwalsh.name/squash-commits-git-->

pranavkarthik10 commented:
@mariobehling Please review and merge! Thanks :)

Dishebh commented:
@pranavkarthik10 pls have a look at the `Schedule Roster` section, hope it doesn't get affected / cut due to this. Thanks.

pranavkarthik10 commented:
@Dishebh Thank you for letting me know! I have fixed it and simplified the code massively by converting the schedule roster section to an image. It still preserves quality in different devices.
<img width="427" alt="Screen Shot 2020-01-01 at 8 44 36 PM" src="https://user-images.githubusercontent.com/44630385/71652072-88a00280-2cd7-11ea-8a20-82c833b46805.png">

pranavkarthik10 commented:
@mariobehling Please review this! Thank you.

pranavkarthik10 commented:
@mariobehling Please review and merge when you get a chance. Thanks!

pranavkarthik10 commented:
@mariobehling May I know why, otherwise the schedule roster section is the only reason why the mobile view does not look good. There are other ways to make it responsive but it would not look as good as the desktop version.
